rubygem-smart_proxy_remote_execution_ssh/rubygem-smart_proxy_remote_execution_ssh.spec

```diff
@@ -35,6 +36,7 @@ BuildRequires: ruby(release)
 %endif
 BuildRequires: rubygems-devel
 BuildArch: noarch
+Obsoletes:  tfm-rubygem(smart_proxy_remote_execution_ssh_core) < 0.1.4
```

Comment:
1. Stray space before `tfm-`
1. On Fedora this has no `tfm-` prefix (`%{?scl:tfm-}`)